src/widget/gui.cpp

```diff
@@ -322,25 +322,34 @@ QString GUI::passwordDialog(const QString& cancel, const QString& body)
 
 void GUI::_clearContacts()
 {
-    Nexus::getDesktopGUI()->clearContactsList();
+    Widget* w = Nexus::getDesktopGUI();
+    if (w)
+        w->clearContactsList();
```

Comment:
1. Absolutely, and that works both ways :) 
We clearly don't stick to this one coding style rule, and it doesn't seem very useful. Why have it?
2. I'm neutral on that issue in general